packages/core/utils/lib/convert-query-params.js

```diff
@@ -215,6 +238,10 @@ const convertNestedPopulate = (subPopulate, schema) => {
     return true;
```

Comment:
Can we remove this if we just do a truthy check?

Reply:
I'm not sure. We want to convert the populate param to DB params, so `true` is better than `*` no?

Reply:
Let me rephrase '*' is a string so it will be converted to true in the isString just below. making this if unnecessary :)

Reply:
oh ok yes :p Let's remove it